<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualSort_Qt_SFMLClass</class>
 <widget class="QMainWindow" name="VisualSort_Qt_SFMLClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>463</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>463</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #36454f;
}

QLabel {
	color: #ffffff;
}

#title {
	font: 30pt bold;
}

QSpinBox {
	background-color: #36454f;
	color: #ffffff;
	selection-background-color: #65816d;
}

QSpinBox::up-button {
	border-top: 1px solid #ffffff;
	border-left: 1px solid #ffffff;
	border-right: 1px solid #ffffff;
	border-bottom: 0.5px solid #ffffff;
	background-color: #36454f;
}

QSpinBox::up-arrow {
	image: url(C:/Users/marem/Downloads/vectorpaint (9).svg);
}

QSpinBox::down-button {
	border-top: 0.5px solid #ffffff;
	border-left: 1px solid #ffffff;
	border-right: 1px solid #ffffff;
	border-bottom: 1px solid #ffffff;
	background-color: #36454f;
}

QSpinBox::down-arrow {
	image: url(C:/Users/marem/Downloads/vectorpaint (10).svg);
}

QComboBox {
	border: 1px solid #ffffff;
	background-color: #36454f;
	color: #ffffff;
	selection-background-color: #36454f;
}

QAbstractItemView {
	background-color: #36454f;
	color: #ffffff;
	selection-background-color: #65816d;
}

QCheckBox {
	color: #ffffff;
}

QCheckBox::indicator {
	border: 1px solid #ffffff;
	background-color: #36454f;
}

QCheckBox::indicator:checked {
	image: url(C:/Users/marem/Downloads/vectorpaint (5).svg);
}

QSlider::groove:horizontal {
	border: 1px solid #ffffff;
    /*border: 1px solid #ffa500;*/
    /*height: 8px;*/
    background: #36454f;/*#ff8243;*/
    margin: 2px 0;
}

QSlider::handle:horizontal {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #ffffff, stop:1 #a0a0a0);
	/*background: #800020;*/
	/*border: 1px solid #5c5c5c;*/
    width: 8px;
    margin: -2px 0;
    /*border-radius: 3px;*/
}

QPushButton {
	border: 1px solid #ffffff;
	background-color: #36454f;
	color: #ffffff;
	border-radius: 5px;
}

QPushButton::hover {
	border: 1px solid #ffffff;
	background-color: #65816d;
	color: #ffffff;
	border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="delay">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="el_num">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="delay_show">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value" stdset="0">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="option_sound">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Sound</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of elements</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Delay (Î¼s)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="sort_combo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <item>
     <property name="text">
      <string>bubble sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>quick sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>selection sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>insertion sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>merge sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>heap sort</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Visual Sort</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VisualSort_Qt_SFML.qrc"/>
 </resources>
 <connections/>
</ui>
